functions/evia-desk-firestore-copy/all_namespaces/all_kinds/all_namespaces_all_kinds.export_metadata

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Do we need these files? I saw we have two folders inside with the name evia-desk-firestore-copy (2). Why do we need this, and why do we have two?